docs/source/cpp/pretrained_models/online_transducer.rst

```diff
@@ -20,6 +20,49 @@ This sections lists models trained using `icefall`_.
 English
 ^^^^^^^
 
+icefall-asr-librispeech-pruned-transducer-stateless7-streaming-2022-12-29
```

Comment:
Could you also update
https://github.com/k2-fsa/sherpa/blob/master/.github/scripts/run-online-transducer.sh